src/builders/polylines.js

```diff
@@ -510,69 +563,81 @@ function addFan (coord, nA, nC, nB, uvA, uvC, uvB, isCap, context) {
 
 //  addBevel    A ----- B
 //             / \     / \
-//           /   /\   /\  \
-//              /  \ /  \  \
+//            /  /\   /\  \
+//           /  /  \ /  \  \
 //                / C \
-function addBevel (coord, nA, nC, nB, uA, uC, uB, context) {
+function addBevel (coord, eA, eC, eB, uA, uC, uB, context) {
     var pivotIndex = context.vertex_data.vertex_count;
 
-    addVertex(coord, nC, uC, context);
-    addVertex(coord, nA, uA, context);
-    addVertex(coord, nB, uB, context);
-
-    var orientation = nA[0] * nB[1] - nA[1] * nB[0] > 0;
+    // ccw or cw bend?
+    var cross = eA[0] * eB[1] - eA[1] * eB[0] > 0;
+
+    if (cross) { // ccw bend
+        // pivot
+        addVertex(coord, Vector.neg(eC), false, uC, context, -1);
```

Comment:
Bevel joins are broken - expected third parameter of `addVertex()` should be the offset vector (line's 2d normal), but it's being set to `false` here (not sure what the correct vector value should be).